server/routes/index.js

```diff
@@ -0,0 +1,40 @@
+const userController = require('../controllers').user;
+const documentController = require('../controllers').document;
+// const roleController = require('../controllers').role;
```

Comment:
You were getting undefined because your `controller/index.js` does not export role.